src/components/molecules/AnnouncementMessage/AnnouncementMessage.scss

```diff
@@ -11,6 +11,10 @@
   width: 500px;
   margin: 0 auto 0 -250px;
 
+  &.center {
+    left: 50%;
+  }
```

Comment:
Why this instead of more standard methods such as `margin` `auto`/etc?

- https://css-tricks.com/centering-css-complete-guide/

Reply:
Which it appears is already being used on line 12, but there is an arbitrary `-250px` on it as well.

My assumption is that that `250px` at one stage represented the width of the sidebar (though doesn't seem to anymore, since we use `20%` in the variable now)